### ISSUE TITLE
PEVerify/ILVerify testhelpers should make sure that verification errors are obtained in English regardless of the current UI culture.

### ISSUE THREAD

**Issue by AlekseyTs** [dotnet/roslyn#71509]
See https://github.com/dotnet/roslyn/pull/71444#discussion_r1442295785 for context

**Comment by AlekseyTs** [comment]
> I think ILVerify uses InvariantCulture for the message

It looks like there was a failure during CI, indicating that something didn't work, The test got disable for non-English environments.  We need to understand what is going on, rather that continuing with disabled tests.

**Comment by jcouv** [comment]
Agreed. Based on the format of the error message in CI, it looks like the problem was with PEVerify.
